bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/Clip2ThingHandler.java

```diff
@@ -0,0 +1,1128 @@
+        if (!disposing) {
+            boolean resourceConsumed = false;
+            String incomingResourceId = resource.getId();
+            if (resourceId.equals(incomingResourceId)) {
+                if (resource.hasFullState()) {
+                    thisResource = resource;
+                    if (!updatePropertiesDone) {
+                        updateProperties(resource);
+                        resourceConsumed = updatePropertiesDone;
+                    }
+                }
+                if (!updateDependenciesDone) {
+                    resourceConsumed = true;
+                    threadPool.execute(() -> updateDependencies());
+                }
+            } else if (ResourceType.SCENE == resource.getType()) {
+                Resource cachedScene = sceneContributorsCache.get(incomingResourceId);
+                if (Objects.nonNull(cachedScene)) {
+                    Setters.setResource(resource, cachedScene);
+                    resourceConsumed = updateChannels(resource);
+                    sceneContributorsCache.put(incomingResourceId, resource);
+                }
+            } else {
+                Resource cachedService = serviceContributorsCache.get(incomingResourceId);
+                if (Objects.nonNull(cachedService)) {
+                    Setters.setResource(resource, cachedService);
+                    resourceConsumed = updateChannels(resource);
+                    serviceContributorsCache.put(incomingResourceId, resource);
+                    if (ResourceType.LIGHT == resource.getType() && !updateLightPropertiesDone) {
+                        updateLightProperties(resource);
+                    }
+                }
+            }
+            if (resourceConsumed) {
+                logger.debug("{} -> onResource() consumed resource {}", resourceId, resource);
+            }
+        }
+    }
+
+    /**
+     * Update the thing internal state depending on a full list of resources sent from the bridge. If the resourceType
+     * is SCENE then call updateScenes(), otherwise if the resource refers to this thing, consume it via onResource() as
+     * any other resource, or else if the resourceType nevertheless matches the thing type, set the thing state offline.
+     *
+     * @param resourceType the type of the resources in the list.
+     * @param fullResources the full list of resources of the given type.
+     */
+    public void onResourcesList(ResourceType resourceType, List<Resource> fullResources) {
+        if (resourceType == ResourceType.SCENE) {
+            updateSceneContributors(fullResources);
+        } else {
+            fullResources.stream().filter(r -> resourceId.equals(r.getId())).findAny()
+                    .ifPresentOrElse(r -> onResource(r), () -> {
+                        if (resourceType == thisResource.getType()) {
+                            logger.debug("{} -> onResourcesList() configuration error: unknown resourceId", resourceId);
```

Comment:
This logging is redundant since the status update below is already logged by the framework.

Reply:
No. See my prior comment concerning filtering the debug messages with the resourceId as prefix.